services/repository/src/test/java/org/sagebionetworks/repo/web/AccessInterceptorTest.java

```diff
@@ -74,6 +75,9 @@ public class AccessInterceptorTest {
 	@Mock
 	TestClock mockClock;
 
+	@Mock
+	AwsKinesisFirehoseLogger firehoseLogger;
+
 	@Captor
 	ArgumentCaptor<AccessRecord> recordCaptor;
 

```

Comment:
Test that the mock is called?

verify(mockFirleHose).calledWithTheExpecteParams()